src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -391,9 +391,13 @@ String generateDeclarations(
       List<SourceFile> sourceFiles, List<SourceFile> externs, Depgraph depgraph)
       throws AssertionError {
     compiler.compile(externs, sourceFiles, opts.getCompilerOptions());
-    precomputeChildLists();
-    collectTypedefs();
-    String dts = produceDts(depgraph);
+    String dts = "";
```

Comment:
It's fine to keep it as is, I think. blaze/bazel will error if you did not produce your output *and* didn't return with an error code. If you return with an error code, it doesn't matter whether you wrote or not.